### ISSUE TITLE
A11y_Ability Summit Phase 1 Digital Venue_Homepage_Video player_Screen reader: Not able to adjust the "Seek slider" of the Video player by swiping one finger up or down.

### ISSUE THREAD

**Issue by PadminiGariki** [shaka-project/shaka-player#8864]
### GitHub Tags:
#A11y; #A11yTCS; #A11yWCAG2.0; #Android; #WCAG2.1; #Full Test Pass; #WCAG1.3.1; #Info and relationships; #Ability Summit 2025; #Ability summit Phase 1; #Microsoft Ability Summit; #A11ySev3;

"[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier." 
### Environment Details: 
Application Name: Ability Summit Phase 1 Digital venue  
#URL: https://aka.ms/AbilitySummitFY25-Phase1 
Android Version: 15 
Screen reader: Talkback 
### Repro Steps: 
1. Hit the URL: https://aka.ms/AbilitySummitFY25-Phase1 
2. Home page will open. 
3. Turn on talkback. 
4. Navigate by swiping right till seek slider. 
5. Now swipe by one finger up or down as per the talkback announcement. 
6. Observe that not able to adjust the "Seek slider" of the Video player by swiping one finger up or down. 
### Actual: 
Not able to adjust the "Seek slider" of the Video player by swiping one finger up or down. 
### Expected: 
We should be able to adjust the "Seek slider" of the Video player by swiping one finger up or down. 
### User Impact: 
Screen reader users will not be able to adjust the "Seek slider" of the Video player by swiping one finger up or down. 
### WCAG Reference: 
[WCAG 1.3.1- Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

Attachments:

[8864.zip](https://github.com/user-attachments/files/21160457/8864.zip)

**Comment by avelad** [comment]
This issue does not follow the structure of issues created for the repository and the attachment does not show anything from Shaka.

**Comment by PadminiGariki** [comment]
Hi @avelad 

#Regressed:07-11-25;

Could you please provide us with the proper structure for creating an issue?
Also, can you please check the attachment below once again.


![Image](https://github.com/user-attachments/assets/b54df415-d5d1-4efd-87ed-0d3d0042cdba)

**Comment by PadminiGariki** [comment]
Hi @avelad 
why the bugs got closed could you please provide us the comments
